deployment/scripts/4-build-db.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+NAMESPACE_NAME="openline"
+FILES="./openline-setup/example_provisioning.sql"
+
+echo "  🦦 Provisioning PostgreSQL"
+
+SQL_USER=openline SQL_DATABABASE=openline SQL_PASSWORD=password
+
+while [ -z "$pod" ]; do
+  pod=$(kubectl get pods -n $NAMESPACE_NAME|grep message-store|grep Running| cut -f1 -d ' ')
+  if [ -z "$pod" ]; then
+    echo "  ⏳ message-store not ready, please wait..."
+    sleep 2
+  fi
+  sleep 2
+done
+
+pod=$(kubectl get pods -n $NAMESPACE_NAME|grep postgresql-customer-os-dev|grep Running| cut -f1 -d ' ')
+
+while [ -z "$pod" ]; do
+  pod=$(kubectl get pods -n $NAMESPACE_NAME|grep postgresql-customer-os-dev|grep Running| cut -f1 -d ' ')
+  if [ -z "$pod" ]; then
+    echo "  ⏳ message-store not ready, please wait..."
+    sleep 2
+  fi
+  sleep 2
```

Comment:
delete this line, best to only sleep if needed instead of doing 2x the sleeping

Reply:
same on line 16 I presume

Reply:
yup